docs/internal-documentation/maintenance/Instructions-for-translators.md

```diff
@@ -337,19 +338,16 @@ is to make a copy of an existing file.
   grep xx_ /etc/locale.gen  # Works in Debian and Ubuntu
   ```
 
-* Go to the *share* directory and use an existing PO file, say sv.po, and make a
-  copy of that to the new file name. Update it and "add" it to `git` before
-  working on it.
+* Go to the *share* directory and create a new blank PO file. Then "add" it
+  to `git` before working on it.
   ```
   cd share
-  cp sv.po xx.po
-  ./update-po xx.po
+  make POLANG=xx new-po
   git add xx.po
   ```
 
-* When you do the update of the new PO file you have to replace all *msgstr*
-  in Swedish with the translation in the new language, but also update the
-  "Language" field in the header.
+* Update the file header, especially the "Content-Type" field with the proper
+  charset.
```

Comment:
The header that I got was
```
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-02-01 21:29+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"
```

I think we should specify what it should be. The headers from fr.po and sv.po look pretty much the same. The sv.po header is
```
msgstr ""
"Project-Id-Version: 1.0.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-01-24 13:17+0000\n"
"PO-Revision-Date: 2023-01-24 13:16+0000\n"
"Last-Translator: mats.dufberg@iis.se\n"
"Language-Team: Zonemaster Project\n"
"Language: sv\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
```

Reply:
Do you have ideas on what to specify for each field? If we define values, they can be automatically set on file creation.

Reply:
We can take the values as they are set in sv.po and fr.po.